RecoTracker/LST/interface/LSTPhase2OTHitsInput.h

```diff
@@ -0,0 +1,47 @@
+#ifndef RecoTracker_LST_LSTPhase2OTHitsInput_h
+#define RecoTracker_LST_LSTPhase2OTHitsInput_h
+
+#include <memory>
+#include <vector>
+
+class LSTPhase2OTHitsInput {
+public:
+  LSTPhase2OTHitsInput();
+  ~LSTPhase2OTHitsInput();
+
+  LSTPhase2OTHitsInput(LSTPhase2OTHitsInput const&) = delete;
+  LSTPhase2OTHitsInput& operator=(LSTPhase2OTHitsInput const&) = delete;
+  LSTPhase2OTHitsInput(LSTPhase2OTHitsInput&&);
+  LSTPhase2OTHitsInput& operator=(LSTPhase2OTHitsInput&&);
+
+  void setLSTPhase2OTHitsTraits(std::vector<unsigned int> ph2_detId,
+                                std::vector<float> ph2_x,
+                                std::vector<float> ph2_y,
+                                std::vector<float> ph2_z) {
+    ph2_detId_ = ph2_detId;
+    ph2_x_ = ph2_x;
+    ph2_y_ = ph2_y;
+    ph2_z_ = ph2_z;
+  }
+
+  std::vector<unsigned int>& ph2_detId() { return ph2_detId_; }
+  std::vector<unsigned int> const& ph2_detId() const { return ph2_detId_; }
```

Comment:
```suggestion
  std::vector<unsigned int> const& detId() const { return detId_; }
```
everything is already clearly phase-2 from the type name. I don't see a good reason to repeat it in every member